.github/workflows/check_and_build.yml

```diff
@@ -2,9 +2,13 @@ name: Check & Build
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
+      - 'release-[0-9].[0-9]-*'
```

Comment:
`release-[0-9].[0-9]-*` doesn't match `release-5.0-rc

Reply:
Oh, it does not match "release-4.0".